app/screens/onboarding/slide.tsx

```diff
@@ -0,0 +1,221 @@
+};
+
+export const ONBOARDING_CONTENT_MAX_WIDTH = 520;
+
+const getStyleSheet = makeStyleSheetFromTheme((theme: Theme) => ({
+    image: {
+        justifyContent: 'center',
+        maxHeight: 180,
+    },
+    title: {
+        color: theme.centerChannelColor,
+        textAlign: 'center',
+        paddingHorizontal: 20,
+        maxWidth: ONBOARDING_CONTENT_MAX_WIDTH,
+        marginBottom: 12,
+    },
+    fontTitle: {
+        marginTop: 32,
+        ...typography('Heading', 1000, 'SemiBold'),
+    },
+    fontFirstTitle: {
+        ...typography('Heading', 1200, 'SemiBold'),
+        paddingTop: 48,
+        letterSpacing: -1,
+    },
+    widthLastSlide: {
+        paddingHorizontal: 50,
+    },
+    firstSlideInitialPosition: {
+        left: 200,
+        opacity: 0,
+    },
+    description: {
+        textAlign: 'center',
+        paddingHorizontal: 20,
+        color: changeOpacity(theme.centerChannelColor, 0.64),
+        ...typography('Body', 200, 'Regular'),
+        maxWidth: ONBOARDING_CONTENT_MAX_WIDTH,
+    },
+    itemContainer: {
+        flex: 1,
+        alignItems: 'center',
+        justifyContent: 'center',
+    },
+}));
+
+const FIRST_SLIDE = 0;
+
+const SlideItem = ({theme, item, scrollX, index, lastSlideIndex}: Props) => {
+    const {width} = useWindowDimensions();
+    const styles = getStyleSheet(theme);
+
+    /**
+     * Code used to animate the first image load
+     */
+    const [firstLoad, setFirstLoad] = useState(true);
+
+    const initialImagePosition = useSharedValue(width);
+    const initialTitlePosition = useSharedValue(width);
+    const initialDescriptionPosition = useSharedValue(width);
+
+    const initialElementsOpacity = useSharedValue(0);
+
+    useEffect(() => {
+        if (index === FIRST_SLIDE) {
+            initialImagePosition.value = withTiming(0, {duration: 600});
+            initialTitlePosition.value = withTiming(0, {duration: 800});
+            initialDescriptionPosition.value = withTiming(0, {duration: 1000});
+
+            initialElementsOpacity.value = withTiming(1, {duration: 1000});
+            setFirstLoad(false);
+        }
+    }, []);
+
+    const translateFirstImageOnLoad = useAnimatedStyle(() => {
+        return {
+            transform: [{
+                translateX: initialImagePosition.value,
+            }],
+            opacity: initialElementsOpacity.value,
+        };
+    });
+
+    const translateFirstTitleOnLoad = useAnimatedStyle(() => {
+        return {
+            transform: [{
+                translateX: initialTitlePosition.value,
+            }],
+            opacity: initialElementsOpacity.value,
+        };
+    });
+
+    const translateFirstDescriptionOnLoad = useAnimatedStyle(() => {
+        return {
+            transform: [{
+                translateX: initialDescriptionPosition.value,
+            }],
+            opacity: initialElementsOpacity.value,
+        };
+    });// end of code for animating first image load
```

Comment:
Do we need this comment? IF so, shouldn't it be after a line break?

Reply:
the comment was more informative probably for me during development, but I still find value on it. I will add the line break. 👍